tests/cases/fourslash/renameJsExports.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path='fourslash.ts'/>
+
+// @allowJs: true
+// @Filename: a.js
+////exports.[|area|] = function (r) { return r * r; }
+
+// @Filename: b.js

```

Comment:
It would be worth adding a cross JavaScript/TypeScript test also. i.e. `a.js` is the same, but with a `b.ts` that uses `import {area} from './a'` syntax.